features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacy.kt

```diff
@@ -161,6 +166,16 @@ enum class SessionReplayPrivacy {
         return mappersList
     }
 
+    private fun buildBase64Serializer(): Base64Serializer {
+        val bitmapPool = BitmapPool()
+        val base64LRUCache = Base64LRUCache()
+
+        val builder = Base64Serializer.Builder()
```

Comment:
Builder methods should be chainable, it's the pattern we've been using in the SDK.
E.g. [Configuration.kt](https://github.com/DataDog/dd-sdk-android/blob/3d566c08b9cabb8d638ae1940be717befef877c0/dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt#L58) allows a use case such as this [SampleApplication.kt](https://github.com/DataDog/dd-sdk-android/blob/3d566c08b9cabb8d638ae1940be717befef877c0/sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt#L175) .

Reply:
I'm modifying the builder in order to pass the dependencies in the builder constructors, so the setters won't exist anymore